nss-sapphire-elephants/WorkforceManagement
# Ticket 13 Complete

# Description

Ticket 13 Complete.
When you click on the computer link, you will now see which employee is assigned to which computer

## Type of change

Please delete options that are not relevant.

# Testing Instructions for Change Made

```git fetch --all```
```git checkout HM&MR_Ticket13```
Check to see that the employee name is displayed
Create a new computer and assign it to an employee through a dropdown

# Checklist:
